test/boost/sstable_compaction_test.cc

```diff
@@ -4660,7 +4660,7 @@ SEASTAR_TEST_CASE(simple_backlog_controller_test) {
         auto manager = compaction_manager(std::move(cfg), as, task_manager);
 
         auto add_sstable = [&env, gen = make_lw_shared<unsigned>(1)] (table_for_tests& t, uint64_t data_size, int level) {
-            auto sst = env.make_sstable(t.schema(), "", (*gen)++, la, big);
+            auto sst = env.make_sstable(t.schema(), env.tempdir().path().native(), (*gen)++, la, big);
```

Comment:
I wonder how did this test have permissions to write to `/` at all?

Reply:
It doesn't, that's the point, and logs are full of errors. So I do wonder too, but this time -- how did this test work at all?

Reply:
Maybe @raphaelsc can shed some light on that. The test is likely broken. But in any case, this PR makes sense so I will queue it. We can fix the test if needed in a follow-up.